tests/sentry/digests/test_utilities.py

```diff
@@ -174,39 +216,15 @@ def setUp(self):
             fallthrough=True,
         )
 
-    def create_event_data(self, filename, url="http://example.com"):
-        data = {
-            "tags": [("level", "error")],
```

Comment:
Is this still needed? I cannot see it between lines 181 and 197

Reply:
It's not needed anymore